scst-store/logs.hbs.md

```diff
@@ -174,6 +178,90 @@ Moving the payload information at this level helps keep the production log outpu
 * Some information in these payloads may be sensitive, and the user may not want them exposed in
 production environment logs.
 
+##  <a id='graphql-endptlog-out'></a> GraphQL endpoint log output (Only applicable to Artifactory Metadata Repository logs)
+
+When an GraphQL endpoint handles a request, the AMR generates two and five log lines. They are:
+
+1. Every request will have received by endpoint will have `Processing request`. This will include operation called and actual fields asked in the response output 
+1. Every response will yield actual query and status
+1. If the endpoint returns a response body, it outputs a second `Request response` line with an extra
+key `payload`, and its value is set to the entire response body. This line is shown at the `debug`
+   verbosity level.
+
+### <a id='graphql-endptlog-out-format'></a>  Format
+
+The logs use JSON output format.
+
+When the AMR handles a request, it outputs some GraphQL endpoint access information in the following
+format:
+
+```console
+{"level":"info","ts":"2023-03-23T13:11:31.161531-06:00","logger":"Artifact Metadata Repository","msg":"Processing request","hostname":"shaileshp2DMD6R.vmware.com","getAllApps":"query getAllApps {\n  apps(latest:true) {\n    \n    timestamp\n    location {\n      alias\n    }\n  }\n}"}
+{"level":"info","ts":"2023-03-23T13:11:31.172953-06:00","logger":"Artifact Metadata Repository","msg":"Request response","hostname":"shaileshp2DMD6R.vmware.com","getAllApps":"query getAllApps {\n  apps(latest:true) {\n    \n    timestamp\n    location {\n      alias\n    }\n  }\n}","code":200,"response":"OK"}
+```
+
+####  <a id='graphql_key-val'></a> Key-value pairs
+
+Since JSON output format uses Key-value pairs, the tables in the following sections list each key and the meaning of their values.
```

Comment:
Key doesn't need to be capitalized. Also maybe reword to something like "The following tables list the meaning of each key found in the logs"